### PR TITLE
feat(web-barbies) : Add new blog post

### PR DESCRIPTION
**Pull request by TsehaynehGetaneh**
# Add new blog post
  - blog post for a user is implemented using RTK Query and local json-server.
  
  please, review the implementation.

newBlog-desktop

### CODE REVIEW

File: Web/starter_project_barbies/components/blog/BlogForm.tsx [hunk 1]
```diff
@@ -1,41 +1,101 @@
...
-  id: string;
+  identity: string;
 }
 
 // File input conponent for file uploads
-const FileInput: React.FC<FileInputProps> = ({ text, id }) => {
+const FileInput: React.FC<FileInputProps> = ({ text, identity }) => {
   return (
-    <label htmlFor={id} className="bg-white text-gray rounded-md px-4 py-2 hover:bg-gray-200">
+    <label htmlFor={identity} className="bg-white text-gray rounded-md px-4 py-2 hover:bg-gray-200">
```
Comment (hello-world-was-taken) [inline Web/starter_project_barbies/components/blog/BlogForm.tsx:18]: we should stick with the convention we are using for the other ids

File: Web/starter_project_barbies/components/blog/BlogForm.tsx [hunk 2]
```diff
@@ -49,9 +109,9 @@ const BlogForm: React.FC = () => {
 
             <div className="flex flex-wrap flex-col md:flex-row gap-2 items-center mt-3">
               <p>Please,</p>
-              <FileInput text="Upload File" id="upload-file" />
+              <FileInput text="Upload File" identity="upload-file" />
```
Comment (hello-world-was-taken) [inline Web/starter_project_barbies/components/blog/BlogForm.tsx:112]: [nit] again lets change them to ids

**Review by hello-world-was-taken — CHANGES_REQUESTED** [review]
Good job overall, one more thing, let's make sure we are running prettier before creating PRs.

**Review by hello-world-was-taken — APPROVED** [review]

**Review by Adona12 — APPROVED** [review]
Reviewed

**Pull request merged**